package.json

```diff
@@ -71,6 +71,7 @@
     "clean-webpack-plugin": "^0.1.17",
     "commander": "^2.9.0",
     "css-loader": "^0.28.7",
+    "express": "^4.16.3",
```

Comment:
We didn't have express already added as a dependency? that seems like a miss.

Reply:
Yeah?? I think since we were only using it for heroku at this point it handled it for us?